### PR TITLE
Makes rads not kill people in stomachs

### PR DESCRIPTION
**Pull request by Cameron653**
Exactly what it says on the tin. Useful for miners/xenoarchs toting people around.

### CODE REVIEW

**Comment by Arokha** [comment]
"I can web edit this real fast!" he said
And now he has to do it again ;o

File: code/datums/repositories/radiation.dm [hunk 1]
```diff
@@ -133,6 +133,7 @@ var/global/repository/radiation/radiation_repository = new()
 
 /mob/living/rad_act(var/severity)
 	if(severity)
-		src.apply_effect(severity, IRRADIATE, src.getarmor(null, "rad"))
+		if(!istype(src.loc, /obj/belly)) //eaten mobs are made immune to radiation //VOREStation Edit
```
Comment (Arokha) [inline code/datums/repositories/radiation.dm]: isbelly(loc)

**Comment by Cameron653** [comment]
...Guh. Let me fix this. Tried to copy & paste this after being given it.

**Comment by Cameron653** [comment]
There we go, thanks for the better proc.

File: code/datums/repositories/radiation.dm [hunk 2]
```diff
@@ -133,6 +133,7 @@ var/global/repository/radiation/radiation_repository = new()
 
 /mob/living/rad_act(var/severity)
 	if(severity)
```
Comment (Arokha) [inline code/datums/repositories/radiation.dm]: Purely cosmetic but might want to combine the two if() statements since there's nothing between them.

Comment (Cameron653) [reply code/datums/repositories/radiation.dm]: Ah, alrighty.

**Pull request merged**